react/FileSelector/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const FileSelector = ({
+  className,
+  disabled,
+  hidden,
+  multiple,
+  processSelected,
+  renderContent,
+  ...inputProps
+}) => (
+  <label role="button" disabled={disabled} className={className}>
+    {renderContent()}
```

Comment:
It's more a `renderLabel` than a `renderContent`, but I'm not very good to find the right name :/

Reply:
I would suggest using `children` here instead. Render props with custom names are cool for rendering a fraction of a component, ie. `renderModalHeader` or whatnot. Here there ins't anything else visible aside from the render prop, so I think `children` would work best.

Reply:
:+1: